packages/core/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## 0.6.12 - not yet released
 
+* `LanguageFactory` instances take care of containment: e.g., creating an entity automatically adds that to the language, and likewise for features (to classifiers) and literals (to enumerations). 
+* Add an object type `SerializationOptions` to configure the `serializeNodes` function.
+    All options (which are all optional) are:
+
+    1. `serializeEmptyValues`: a boolean flag that determines whether empty (unset) feature values are explicitly serialized or skipped during serialization.
+        This potentially reduces the size of the serialization substantially, helping with performance.
+        The default value is `true`, meaning that empty values – currently of containment and reference features – are explicitly serialized.
```

Comment:
> currently of containment and reference features

This is not according to spec, do you want to explain? Or also create a flag for it?

Reply:
No, I made this uniform/consistent: see latest commit.